fairroot/base/source/FairMixedSource.h

```diff
@@ -43,11 +43,9 @@ class FairMixedSource : public FairFileSourceBase
                     const char* Title = "InputRootFile",
                     UInt_t identifier = 0);
     //  FairMixedSource(const FairMixedSource& file);
-    ~FairMixedSource() override;
```

Comment:
Please keep all the other (`=defaulted`) ctors as well. :-)

Reply:
Can you please re-introduce all those removed dtors again? Should even reduce the diff-size. :wink:

Reply:
I think I did with 
```c++
~FairMixedSource() override = default;
```
Isn't this what you suggested?

Reply:
I also meant the other ones. Like

```cpp
~PixelDigiSource() override = default;
~FairRootFileSink() override = default;
FairFileSource::~FairFileSource() = default;
```

Reply:
Ok, I have changed according to your request.